lib/board.rb

```diff
@@ -0,0 +1,18 @@
+class Board
+    attr_reader :cells
+
+    def initialize(cells)
```

Comment:
This will need refactoring as you said, I will make modifications for my part in the process.